herchel/siervo.py

```diff
@@ -0,0 +1,117 @@
+class GradeManagementSystem:
+    def __init__(self):
+        self.students = {}  
+        self.subjects = [] 
+        self.max_grade = 100 
+
+    def add_grade(self):
+        name = input("Enter the student's name: ").strip().lower()
+        if name not in self.students:
+            self.students[name] = {}
+        subject = input("Enter the subject: ").strip().lower()              
+        if subject not in self.subjects:
+            self.subjects.append(subject)
+        
+        try:
+            grade = float(input(f"Enter the grade for {subject}: "))
+            if 0 <= grade <= self.max_grade:
+                self.students[name][subject] = grade
+                print(f"Grade {grade} added for {name} in {subject}.")
+            else:
+                print(f"Invalid. Must be between 0 and {self.max_grade}.")
+        except ValueError:
+            print("Invalid input. Grade must be a number.")
+
+    def view_grades(self):
+        name = input("Enter the student's name: ").strip().lower()
+        if name not in self.students:
+            print("Student not found.")
+            return
+        print(f"Grades for {name}:")
+        grades = self.students[name]
+        
```

Comment:
remove line 32 space.

Reply:
remove unnecessary space, fix in commit 6bfb109 .